client/src/components/TextArea/Text.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import './Text.css'
+
+const Text = (props) => <p>{props.children}</p>
```

Comment:
Can you use the text area input here? someone should be able to type into this text box. Also, for events, don't forget you need an `onChange` handler so the input can be controlled